draft-barnes-acme.md

```diff
@@ -133,9 +133,6 @@ Subject Public Key:
 Account Key Pair:
 : A key pair for which the ACME server considers the holder of the private key authorized to manage certificates for a given identifier.  The same key pair may be authorized for multiple identifiers.
 
-Recovery Token:
```

Comment:
@martinthomson only if the target actually performs Recovery at some point.  But you're right, and for that reason we either need to make sure we walk Recovery back from "you recover all authorizations" to "you get to reperform DV without having to do Proof of Possession" or do the asymmetric crypto as you suggest.